questions/simple.ts

```diff
@@ -130,6 +132,12 @@ const convertAnswers = async (
       dataSources: [],
       modelConfig: MODEL_GPT4o,
     },
+    financial_report: {
+      template: "multiagent",
```

Comment:
don't delete `examples` instead move it to `templates/components/agents/blog` and copy the new agents to ``templates/components/agents/financial_report`.
then pick the right agents depending on this selection:
```suggestion
      template: "multiagent",
      agents: "financial_report"
```
also change e2e test to test that `financial_report` is compiling